### ISSUE TITLE
[Bug Report] V-data-table does not support keyboard commands for navigating through table headers and rows

### ISSUE THREAD

**Issue by mgren21** [vuetifyjs/vuetify#9289]
### Environment
**Vuetify Version:** 2.1.1
**Last working version:** 1.5.18
**Vue Version:** 2.6.10
**Browsers:** Google Chrome, Mozilla Firefox
**OS:** Windows 10, Linux

### Steps to reproduce
1. Open code pen showing v-data-table in Vuetify 2.1.1
2. Use Tab button to attempt to navigate the table headers and rows

### Expected Behavior
Once tabbing into the table you should be able to navigate through the column headers and select which column(s) to sort by. You should also be able to tab through each row and select the row(s) using enter. 

### Actual Behavior
Hitting the tab button skips over the table entirely and goes straight to the pagination options below the table. Both the table headers and rows cannot be navigated with any keyboard commands.

### Reproduction Link
<a href="https://codepen.io/mgren21/pen/oNNgKJE?&editable=true&editors=101" target="_blank">https://codepen.io/mgren21/pen/oNNgKJE?&editable=true&editors=101</a>

### Other comments
Here is the codepen for 1.5.18 which shows the keyboard commands for selecting header columns for sorting, and for selecting rows in the table working. 
https://codepen.io/mgren21/pen/qBBEezp?&editable=true&editors=101

**Comment by UglyHobbitFeet** [comment]
FYI you can make custom css per theme.  https://stackoverflow.com/questions/58067386/vuetify-themes-with-custom-css

**Comment by joswald76** [comment]
This could be like this issue where the roles are missing when the data-table is generated:
https://github.com/vuetifyjs/vuetify/issues/9060

**Comment by bad4iz** [comment]
Duplicate of  #9164

**Comment by flpehr0** [comment]
> Duplicate of #9164

Hi, it is #9164 that should have been closed in favor of this ticket as accessibility functionality that was previously working and now is not is NOT a feature request. It is a bug. The bug ticket should remain open and the feature request closed.
